pkg/volume/local_volume/local_volume.go

```diff
@@ -201,6 +202,34 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return err
 	}
 
+	// If fsGroup is not nil, we need to make sure the requested fsGroup doesn't
+	// conflict with fsGroup already set for previous pod.
+	if fsGroup != nil {
```

Comment:
The behavior I'm concerned with is that the 2nd pod will write as root, and the 1st pod will not be able to read it.